src/style.css

```diff
@@ -1,3 +1,62 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.link {
+  display: flex;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  align-items: center;
+  height: 100%;
+  white-space: nowrap;
+}
+
+a.link.is-active {
+  color: var(--blue-color) !important;
+  text-decoration: none;
+  background-image: url(./images/Rectangle\ 2.svg);
+  background-repeat: no-repeat;
+
+  background-position: bottom center;
+  background-size: contain;
+}
+
+a.link.is-active:hover,
+a.link.hover:hover {
```

Comment:
The 'hover' class is used here in conjunction with ':hover'. Ensure that this class is defined in your HTML and is intended for this specific styling. If it is meant to apply to all links, consider using a more general selector or applying the class consistently.